pkg/controllers/dynakube/connectioninfo/oneagent/reconciler.go

```diff
@@ -128,6 +128,11 @@ func (r *reconciler) reconcileConnectionInfo(ctx context.Context) error {
 		return err
 	}
 
+	r.dk.Status.OneAgent.ConnectionInfoStatus.TenantTokenHash, err = hasher.GenerateHash(connectionInfo.ConnectionInfo.TenantToken)
+	if err != nil {
+		return err
```

Comment:
```suggestion
		return errors.Wrap(err, "failed to generate TenantTokenHash")
```
just a little bit more context, please ;)